datafusion/src/datasource/file_format/csv.rs

```diff
@@ -35,25 +35,69 @@ use crate::physical_plan::Statistics;
 
 /// Character Separated Value `FileFormat` implementation.
 pub struct CsvFormat {
+    has_header: bool,
```

Comment:
The doc strings seem to have been lost from this structure

Reply:
My understanding is that the docstrings are useful on public fields only: private field documentation will not be used by the HTML generator anyway, they appear as `/* fields omitted */` (exemple [here](https://docs.rs/arrow/5.5.0/arrow/record_batch/struct.RecordBatch.html)). This is why I moved the documentation to the public `with_xxx` builder methods. Is there a more idiomatic way to do this? Should we leave these fields public?